.github/workflows/2x-kubernetes-version-matrix-test.yaml

```diff
@@ -0,0 +1,70 @@
+name: 2.x Kubernetes Version Matrix Tests
+
+on:
+  push:
+    branches:
+      - 'main'
+      - 'next'
+
+jobs:
+  test-dbless:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
```

Comment:
Can we make `dbless`/`postgres` matrix fields as well? I don't see a reason for duplication here, but maybe I haven't read this diligently enough.

Reply:
Checked over [the documentation](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idstrategymatrix) and got this patched. I don't think it looks as nice:

![Screenshot_2021-07-22_08-37-50](https://user-images.githubusercontent.com/5332524/126640304-f3dd456c-9b6f-4ee2-82e1-f2f26ca498a3.png)

But it's functional and `does-the-right-thing.go` so :+1: